File: Domains/ApplicationDomain/BOA/Entities/Cow.cs
using ApplicationDomain.Core.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace ApplicationDomain.BOA.Entities
{
    public class Cow : EntityBase<int>
    {
        public int ByreId { set; get; }
        public Byre Byre { set; get; }
        public int? MotherId { set; get; }
        public int? FatherId { set; get; }
        public string Name { set; get; }
        public string Status { get; set; }
        public string Code { set; get; }
        public string TinhCode { get; set; }
        public DateTime Birthday { set; get; }
        public string Gender { set; get; }
        public DateTime WeaningDate { set; get; }
        public int FoodSuggestionId { set; get; }
        public FoodSuggestion FoodSuggestion { set; get; }
    }
}

File: Applications/WebAdminApplication/ClientApp/src/app/modules/farmer/models/cow.ts
export class Cow {
  id: number;
  byreId: number;
  code: string;
  name: string;
  motherId: number;
  fatherId: number;
  qrCode: string;
  status: string;
  tinhCode: string;
  birthDay: Date;
  ageNumber: number;
  gender: string;
  weaningDay: Date;
  foodSuggestionId: number;

  constructor() {
    this.id = 0;
    this.byreId = 0;
    this.code = '';
    this.name = '';
    this.motherId = 0;
    this.fatherId = 0;
    this.qrCode = '';
    this.status = '';
    this.tinhCode= '';
    this.birthDay = new Date('02/02/2020');
    this.ageNumber=0;
    this.gender='';
    this.weaningDay= new Date('02/02/2020');
    this.foodSuggestionId = 0;
  }
}
